src/main/steps/applicant1/hub-page/content.ts

```diff
@@ -42,12 +43,12 @@ const en = ({ isDivorce, userCase, referenceNumber, partner, isJointApplication,
     line3: `You do not need to come to the hearing, unless you want to object. You must contact the court by ${dayjs(
       userCase.coDateAndTimeOfHearing
     )
-      .subtract(7, 'day')
+      .subtract(config.get('dates.contactCourtBeforeHearingDays'), 'days')
```

Comment:
the docs say its singular day and not days
https://day.js.org/docs/en/manipulate/add

dayjs().add(7, 'day')